src/hooks/usePelcro/usePelcro.test.js

```diff
@@ -306,6 +358,7 @@ describe("Actions", () => {
 
   describe("switchToAddressView", () => {
     test("switch to address creation view when the user doesn't have any address", () => {
+      console.log("useerrrrr", Pelcro.user.read());
```

Comment:
I think you don't need this line anymore

Reply:
LOL thanks